src/components/Navbar/WalletConnectionAlert.tsx

```diff
@@ -54,23 +58,10 @@ const WalletConnectionAlert: FC<{
   }
 
   return (
-    <Alert
-      status="warning"
-      variant="solid"
-      position="absolute"
-      w="fit-content"
-      paddingRight="40px"
-      top="94px"
-      right="5.25rem"
-    >
+    <Alert status={alertStatus} variant="solid">
       <AlertIcon />
```

Comment:
Let's add `w="fit-content"` to the `Alert` component here and `alignSelf="center"` to the `AlertIcon` component (for some reason we set the `alginSelf` to `flex-start` in Chakra theme in `@threshold-network/components`):

```suggestion
    <Alert status={alertStatus} variant="solid" w="fit-content">
      <AlertIcon alignSelf="center" />
```

Reply:
Addressed in [apply styling to alert related components](https://github.com/threshold-network/token-dashboard/pull/736/commits/39e6e2d006412856ad90c9c80e4f9d712ea3d6e6)